{"name": "针对异常登陆的测试", "status": "passed", "steps": [{"name": "步骤1：登录", "status": "passed", "start": 1609858631986, "stop": 1609858631987}], "attachments": [{"name": "log", "source": "87d25f73-15ca-4215-882e-ed2d72b2de4b-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'name': '登录功能-异常登录-用户名为空', 'username': '', 'password': '654321'}"}], "start": 1609858631983, "stop": 1609858632493, "uuid": "35383b8f-beb6-4cef-9ec6-0322e33ae332", "historyId": "d93c8b521d1ee7fc4eb27338522fd359", "testCaseId": "130e6dce60bc4ef9d882c5f0465dc1d6", "fullName": "cases.test_login.TestLogin#test_login_failed", "labels": [{"name": "story", "value": "登录模块-异常登陆"}, {"name": "epic", "value": "金融界web自动化自测"}, {"name": "feature", "value": "金融界web登录模块"}, {"name": "tag", "value": "@pytest.mark.usefixtures('refresh_page')"}, {"name": "parentSuite", "value": "cases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "PC-20200718ELAI"}, {"name": "thread", "value": "47204-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.test_login"}]}